files/config/kubelet-worker.yaml.tmpl

```diff
@@ -2,7 +2,7 @@ kind: KubeletConfiguration
 apiVersion: kubelet.config.k8s.io/v1beta1
 address: ${DEFAULT_IPV4}
 port: 10250
-healthzBindAddress: ${DEFAULT_IPV4}
+healthzBindAddress: "0.0.0.0"
```

Comment:
It would be good to drop a comment why we are doing so. Otherwise someone will revert it one day because they will think it's unnecessary. Speaking from experiences: https://github.com/giantswarm/k8scloudconfig/pull/742#discussion_r444879887